src/shared-components/button/style.js

```diff
@@ -118,6 +118,7 @@ export const baseButtonStyles = ({
   buttonType,
   isLoading,
```

Comment:
![image](https://user-images.githubusercontent.com/28582716/60453004-c61aba80-9be4-11e9-9510-dec162b4247b.png)

Is it possible to move the loading spinner into the same area as the text here?  It currently pushes way out to the side.